Menu/ContextMenu: add Scrolling demo descriptions (#27315)
DevExpress/DevExtreme: apps/demos/Demos/Menu/Scrolling/description.md

> The Menu component supports submenu item scrolling. If the combined item length exceeds the screen size, a scrollbar appears, and the submenu height is restricted.

Reviewer: ```suggestion
The ContextMenu component supports submenu item scrolling. If the combined item height exceeds the screen size or a pre-defined height limit, a scrollbar appears.
```

> The Menu component supports submenu item scrolling. If the combined item length exceeds the screen size, a scrollbar appears, and the submenu height is restricted.

Author: ```suggestion
The Menu component supports submenu item scrolling. If the combined item height exceeds the screen size or a pre-defined height limit, a scrollbar appears.
```

You can use one of the following event handlers to configure submenus:

- [onSubmenuShowing](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuShowing)

- [onSubmenuShown](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuShown) 

- [onSubmenuHiding](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuHiding)

- [onSubmenuHidden](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuHidden)

> These handlers access `submenuContainer`, the root submenu element, and `itemData`, which contains root item and submenu item data.

Reviewer: If "submenuContainer" and "the root menu element" are the same thing, then I would made the following change.

(Right now, it sounds like `submenuContainer` and "the root submenu elements" are two different items.)

```suggestion
These handlers can access the root submenu element (`submenuContainer`), as well as data from root and submenu items (`itemData`).
```
